kuma/users/tests/test_helpers.py

```diff
@@ -19,3 +21,18 @@ def test_public_email():
     assert ('<span class="email">'
             '&#110;&#111;&#116;&#46;&#97;&#110;&#46;&#101;&#109;&#97;&#105;'
             '&#108;</span>' == public_email('not.an.email'))
+
+
+@pytest.mark.parametrize('case', ('upper', 'lower'))
+def test_is_username_taken_existing_user(wiki_user, case):
+    """
+    Test is_username_taken for an existing user, and also that the username is
+    case insensitive.
+    """
+    assert is_username_taken(getattr(str, case)(wiki_user.username))
+
+
+@pytest.mark.parametrize('username', ('nonexistent', None))
+def test_is_username_taken_nonexisting_user(wiki_user, username):
+    """Test is_username_taken for an non-existing user."""
+    assert not is_username_taken(username)
```

Comment:
These two tests above are not wrong but they're pretty verbose too. It could be simplified to just:
```python
def test_is_username_taken(wiki_user):
    assert is_username_taken(wiki_user.username)
    assert is_username_taken(wiki_user.username.upper())  
    assert not is_username_taken('nonexistent')  
```

Reply:
Yes, that's fine too. In fact, I originally wrote the test that way. Either way is fine to me, but I'm happy to move back to this since the function being tested is so simple.